x-pack/plugins/observability_onboarding/public/assets/standalone_agent_setup.sh

```diff
@@ -19,41 +63,40 @@ updateStepProgress() {
     --no-progress-meter
 }
 
-echo "Downloading Elastic Agent archive"
+ELASTIC_AGENT_DOWNLOAD_URL="https://artifacts.elastic.co/downloads/beats/elastic-agent/${artifact}.tar.gz"
+echo "Downloading Elastic Agent archive from ${ELASTIC_AGENT_DOWNLOAD_URL}"
 updateStepProgress "ea-download" "loading"
-ELASTIC_AGENT_DOWNLOAD_URL="https://artifacts.elastic.co/downloads/beats/elastic-agent/elastic-agent-${ELASTIC_AGENT_VERSION}-linux-x86_64.tar.gz"
 curl -L -O $ELASTIC_AGENT_DOWNLOAD_URL --fail --continue-at -
 if [ "$?" -eq 0 ]; then
   echo "Downloaded Elastic Agent"
   updateStepProgress "ea-download" "complete"
 else
   updateStepProgress "ea-download" "danger"
-  echo "Failed to download Elastic Agent"
-  exit 1
+  fail "Failed to download Elastic Agent"
 fi
 
 echo "Extracting Elastic Agent"
 updateStepProgress "ea-extract" "loading"
-tar -xzf elastic-agent-${ELASTIC_AGENT_VERSION}-linux-x86_64.tar.gz --checkpoint=.1000
+tar -xzf ${artifact}.tar.gz --checkpoint=.1000
 echo ""
 if [ "$?" -eq 0 ]; then
   echo "Elastic Agent extracted"
   updateStepProgress "ea-extract" "complete"
 else
   updateStepProgress "ea-extract" "danger"
-  exit 1
+  fail "Failed to extract Elastic Agent"
 fi
 
 echo "Installing Elastic Agent"
 updateStepProgress "ea-install" "loading"
-cd elastic-agent-${ELASTIC_AGENT_VERSION}-linux-x86_64
+cd ${artifact}
 ./elastic-agent install -f
 if [ "$?" -eq 0 ]; then
   echo "Elastic Agent installed"
```

Comment:
That's not necessarily related to this PR but I get the following, although the agent status check failed cc @ogupte 

```
Elastic Agent has been successfully installed.
Elastic Agent installed
Checking Elastic Agent status
.Error: failed to communicate with Elastic Agent daemon: rpc error: code = Unavailable desc = connection error: desc = "transport: Error while dialing dial unix /var/run/elastic-agent.sock: connect: no such file or directory"
For help, please see our troubleshooting guide at https://www.elastic.co/guide/en/fleet/8.8/fleet-troubleshooting.html
..
Elastic Agent running
Download and save configuration to /opt/Elastic/Agent/elastic-agent.yml
Done with standalone Elastic Agent setup for custom logs. Make sure to add your configuration to /opt/Elastic/Agent/elastic-agent.yml, then look for streaming logs to arrive in Kibana
```